packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -225,7 +225,7 @@ describe('SelectedNetworkController', () => {
     });
   });
 
-  describe('It updates domain state when the network controller state changes', () => {
+  describe('networkController:stateChange and useRequestQueuePreference is true', () => {
```

Comment:
Nit: We may want a test for the opposite condition (i.e. when useRequestQueue is false)

Reply:
done here https://github.com/MetaMask/core/pull/4388/commits/fd79acf9f9b4e22533b66680274e2992bef6afd0